src/components/ExitButton.tsx

```diff
@@ -4,6 +4,6 @@ import { IoCaretBack } from 'react-icons/io5';
 export function ExitPage() {
     const navigate = useNavigate();
     return (
-        <button onClick={() => navigate('/')} className="top-left" style={{position:"absolute", top:"5%", left:"2.5%", fontSize:"30px"}}> <IoCaretBack style={{display: "inline-block"}}/>Exit</button>
+        <button onClick={() => { localStorage.clear(); navigate('/');}} className="top-left" style={{position:"absolute", top:"5%", left:"2.5%", fontSize:"30px"}}> <IoCaretBack style={{display: "inline-block"}}/>Exit</button>
```

Comment:
won't this cause issues when tracking the stats? Probably just clear the qData field instead

Reply:
Oh yeah it will, i'll go and edit it